array-map/main.js

```diff
@@ -0,0 +1,17 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+console.log(numbers.map(x => x*2));
+
+console.log(numbers.map(x => `$${x}.00`))
```

Comment:
This will work with whole number but not if the number already has a decimal. A better solution would be to do something like `x => '$' + x.toFixed(2)`